site/docs/releases.md

```diff
@@ -77,6 +77,15 @@ Apache Iceberg 1.4.3 was released on December 27, 2023. The main issue it solves
 - Core: Expired Snapshot files in a transaction should be deleted by [@bartash](https://github.com/bartash) in [#9223](https://github.com/apache/iceberg/pull/9223)
 - Core: Fix missing delete files from transaction by [@nastra](https://github.com/nastra) in [#9356](https://github.com/apache/iceberg/pull/9356)
 
+## Schedule
```

Comment:
How about just mentioning that 
- Iceberg community aims at providing quarterly releases for minor versions.
- Point releases for blocking issues can be made with priority immediately without waiting for release cycle. 
and mention that previous release timelines can be obtained from https://github.com/apache/iceberg/releases


Because these actual dates may create stress and if we screwup one release date, we have to change the release dates of other releases.  Plus we may directly go to 2.0 after 1.6.0 etc.

Reply:
Open source...open release schedule 😉

Reply:
Fair enough, let me update that way.